.github/workflows/javadoc.yml

```diff
@@ -29,4 +29,14 @@ jobs:
         run: ./mvnw -Dsurefire.useFile=false -DskipTests -Denforcer.skip=true clean install javadoc:aggregate -pl '!unit-tests,!serialization-tests,!jcstress-tests,!junit-trait-runner,!unit-tests-java8,!test-coverage,!p2-repository,!org.eclipse.collections:org.eclipse.collections' --no-transfer-progress
         env:
           MAVEN_OPTS: "-Xmx1g"
+      - name: checks
+        run: |
+          ls -ltr ${{ github.workspace }}/target
+          ls -ltr ${{ github.workspace }}/target/site
+          ls -ltr ${{ github.workspace }}/target/site/apidocs
+      - name: Upload Javadoc
```

Comment:
I added this in case anyone wants to see the generated javadoc given this PR has related javadoc plugin changes. I will revert this section before the PR gets merged.

Reply:
if you have verified that the javadocs look ok, We should be fine 😄